src/cards/stats-card.js

```diff
@@ -34,8 +34,9 @@ const createTextNode = ({
   showIcons,
   shiftValuePos,
   bold,
+  short_values,
 }) => {
-  const kValue = kFormatter(value);
+  const kValue = short_values ? kFormatter(value) : value;
```

Comment:
This is a breaking change for every github readme stats user. 

Current behaviour is to add the `k` at the end. But this will make it so that numbers are not K formatted by default.

Reply:
We should set this parameter `true` by default, and users can opt out by setting `false` 

I recommend a more specific naming for this - `number_format="long"`, `number_format="short"`

Reply:
I already set it to `true` when passing it into `renderStatsCards`. Therefore, there is no breaking change

Reply:
@CaedenPH I will not change my review or remove the `hacktoberfest-accepted` label since I think that will infer with your Hacktoberfest submissions. I, however, think the @anuraghazra naming suggestion is better than the one we came up with. 👍🏻

Reply:
> I recommend a more specific naming for this - `number_format="long"`, `number_format="short"`

I like this idea and I will implement it when I get a chance. Do you think short and long should he the only options?